src/test/java/gov/usgs/wma/statistics/logic/MonthlyStatisticsTest.java

```diff
@@ -157,8 +173,9 @@ public void test_monthlyStats_yearly_sameMonth() throws Exception {
 		List<Value> sorted = new LinkedList<>(samples);
 		StatisticsCalculator.sortByValueOrderAscending(sorted);
 
+		builder.mediation(MediationType.AboveDatum);
 		// we are not testing this method so mock it to return what we need
-		MonthlyStatistics<Value, MediationType> mockstats = new MonthlyStatistics<Value, MediationType>(MediationType.AboveDatum) {
+		MonthlyStatistics<Value> mockstats = new MonthlyStatistics<Value>(builder) {
 //			protected boolean doesThisMonthQualifyForStats(List<Value> monthSamples) {
 //				return monthSamples.size()>0;
 //			}
```

Comment:
Delete comments.

Reply:
Oddly, I think the next PR will clear this.